### ISSUE TITLE
OTU filtering before normalization

### ISSUE THREAD

**Issue by FelipeMSD** [malucalle/Microbiome-Variable-Selection#2]
Dear Dr. Calle and team,

I am working with microbiome analyses in my PhD project and as I saw on this pipeline that you worked with an OTU table that was firstly filtered, resulting in 558 taxa and 47 samples I was wondering what criteria you used for these filters and what would be the impact of this step on the normalization process. Did you remove OTUs based on "relative abundance" and/or prevalence thresholds? I've been talking to people who told me that this filtering should be done after normalization and some that assume either ways are fine, which is making me a little bit confused as I am not an expert in biostatistics.

Thank you very much in advance for your attention.

Kind regards,
Felipe

**Comment by EvaYiwenWang** [comment]
Hey Felipe,

I am very sorry for the late reply. In my analysis, I filtered out the samples with overall counts less than 10, and the variables with overall counts less than 0.01% of the total counts. The detailed explanation can be found here: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0160169 <https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0160169>. I removed OTU based on the counts, and this way is also highly recommended. I have never filtered the data after normalisation, otherwise, I think I would lose some information. I hope my answers are helpful.

Best regards,
Eva (Yiwen)
